_includes/tutorials/joining-stream-stream/ksql/code/src/statements.sql

```diff
@@ -0,0 +1,25 @@
+CREATE STREAM ORDERS (ORDER_ID INT, ORDER_TS VARCHAR, TOTAL_AMOUNT DOUBLE, CUSTOMER_NAME VARCHAR)
+    WITH (KAFKA_TOPIC='_orders',
+          VALUE_FORMAT='JSON',
```

Comment:
We should stick with Avro for these examples since we're nudging people to get the power out of schemas.